core/src/main/java/org/apache/carbondata/core/metadata/SegmentFileStore.java

```diff
@@ -1228,7 +1228,7 @@ public static void removeTempFolder(Map<String, FolderDetails> locationMap, Stri
       locationMap = new HashMap<>();
     }
 
-    SegmentFile merge(SegmentFile mapper) {
+    public SegmentFile merge(SegmentFile mapper) {
```

Comment:
name `mapper` is not good, can you rename it?

Reply:
done